R/data_danica.R

```diff
@@ -0,0 +1,119 @@
+#' NBA Raptor
+#' 
+#' The raw data behind the story 
+#' "Introducing RAPTOR, Our New Metric For The Modern NBA"
+#' \url{https://https://fivethirtyeight.com/features/introducing-raptor-our-new-metric-for-the-modern-nba}
```

Comment:
Fix url

Reply:
Nice catch: you've got `https://` in there twice!